### PR TITLE
added /mytop cmd

### LINKED ISSUE #210: A /mytop command to show user's rank.

### PR DESCRIPTION
**Pull request by coool9**
Closes #210

### CODE REVIEW

**Comment by coool9** [comment]
@Southclaws merge

**Comment by NotUnlikeTheWaves** [comment]
Looks good, maybe it would be nice to also show the amount of messages you've posted aside of the rank?

File: storage/chat.go [hunk 1]
```diff
@@ -77,6 +77,37 @@ func (m *MongoStorer) GetTopMessages(top int) (result TopMessages, err error) {
 	return
 }
 
+func (m *MongoStorer) GetUserMessageCount(discordUserID string) (messageCount int, err error) {
+	messageCount, err = m.chat.Find(bson.M{"discorduserid": discordUserID}).Count()
```
Comment (NotUnlikeTheWaves) [inline storage/chat.go:81]: Curious if this'll be slow for people with many messages, but if that's the case we'll have to try a new pull request then

**Comment by NotUnlikeTheWaves** [comment]
Nice job!

**Pull request merged**